serveradmin/common/templates/base.html

```diff
@@ -15,6 +15,7 @@
 <script type="text/javascript" src="{{ STATIC_URL }}common.js"></script>
 {% block additional_scripts %}
 {% endblock %}
+<link rel="search" type="application/opensearchdescription+xml" href="{{ STATIC_URL }}opensearch.xml" />
```

Comment:
If the file doesn't exist nothing shouldn't break

Reply:
I don't understand.  Why the file wouldn't exist?  You place it there.